app/src/main/java/de/danoeh/antennapod/activity/VideoplayerActivity.java

```diff
@@ -480,4 +484,102 @@ void stop() {
+            case KeyEvent.KEYCODE_M:
+                //Mute/Unmute
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+                    audioManager.adjustStreamVolume(AudioManager.STREAM_MUSIC, AudioManager.ADJUST_TOGGLE_MUTE, 0);
+                } else {
+                    if (audioManager.getStreamVolume(AudioManager.STREAM_MUSIC) == 0) {
+                        //Unmute
+                        audioManager.setStreamVolume(AudioManager.STREAM_MUSIC,
+                                audioManager.getStreamMaxVolume(AudioManager.STREAM_MUSIC), 0);
+                    } else {
+                        //Mute
+                        audioManager.setStreamVolume(AudioManager.STREAM_MUSIC, 0, 0);
+                    }
+                }
+                break;
+            case KeyEvent.KEYCODE_DPAD_UP:
+                //Raise volume
+                audioManager.adjustStreamVolume(AudioManager.STREAM_MUSIC,
+                        AudioManager.ADJUST_RAISE, AudioManager.FLAG_SHOW_UI);
+                break;
+            case KeyEvent.KEYCODE_DPAD_DOWN:
+                //Raise volume
+                audioManager.adjustStreamVolume(AudioManager.STREAM_MUSIC,
+                        AudioManager.ADJUST_LOWER, AudioManager.FLAG_SHOW_UI);
+                break;
+            case KeyEvent.KEYCODE_ESCAPE:
+            case KeyEvent.KEYCODE_F:
+                //Exit fullscreen mode
+                onOptionsItemSelected(new ActionMenuItem(this, 0, android.R.id.home, 0, 0, ""));
+                break;
+            case KeyEvent.KEYCODE_P:
+                //Toggle picture-in-picture mode
+                compatEnterPictureInPicture();
+                break;
+            //Go to x% of video:
+            case KeyEvent.KEYCODE_0:
+                videoPercent = 0f;
+                break;
+            case KeyEvent.KEYCODE_1:
+                videoPercent = 0.1f;
+                break;
+            case KeyEvent.KEYCODE_2:
+                videoPercent = 0.2f;
+                break;
+            case KeyEvent.KEYCODE_3:
+                videoPercent = 0.3f;
+                break;
+            case KeyEvent.KEYCODE_4:
+                videoPercent = 0.4f;
+                break;
+            case KeyEvent.KEYCODE_5:
+                videoPercent = 0.5f;
+                break;
+            case KeyEvent.KEYCODE_6:
+                videoPercent = 0.6f;
+                break;
+            case KeyEvent.KEYCODE_7:
+                videoPercent = 0.7f;
+                break;
+            case KeyEvent.KEYCODE_8:
+                videoPercent = 0.8f;
+                break;
+            case KeyEvent.KEYCODE_9:
+                videoPercent = 0.9f;
+                break;
+            default:
+                return super.onKeyUp(keyCode, event);
+        }
+
+        if (videoPercent != null) {
+            controller.seekTo((int) (videoPercent * controller.getDuration()));
+        }
+        return true;
```

Comment:
I think this could lead to problems. If AntennaPod swallows key events, standard system keys could no longer work (things like copy-paste?)

Reply:
For unhandled keycodes the `default` case is called which calls `super.onKeyUp(keyCode, event);` (l. 573)